client/src/views/Simulation/components/ModelPanel.vue

```diff
@@ -15,7 +15,7 @@
     <global-graph
       v-if="graph"
       :data="graph"
-      :edited-nodes="editedNodes"
+      :edited-nodes="displayedNodes"
```

Comment:
you might change the name of the prop as well from `edited-nodes` to `displayed-nodes` on the `<global-graph>` component.

Reply:
Thanks for the suggestion, updated.